index.html

```diff
@@ -223,7 +223,7 @@
       </section>
       <section>
         <h2>
-          <code>ScreenOrientation</code> interface
+          <code><dfn>ScreenOrientation</dfn></code> interface
```

Comment:
you don't need the code elements here.

Reply:
@marcoscaceres I tried not to remove anything that was already in the spec. Should I do it in follow-up PR or you want to revert this one?

Reply:
A new new PR would be great. Really appreciate your time and help here.